src/Html.php

```diff
@@ -862,17 +824,20 @@ public static function dropDownList($name, $selection = null, $items = [], $opti
      *   mode, we can still obtain the posted unselect value.
      * - encodeSpaces: bool, whether to encode spaces in option prompt and option value with `&nbsp;` character.
      *   Defaults to false.
-     * - encode: bool, whether to encode option prompt and option value characters.
-     *   Defaults to `true`.
+     * - encode: bool, whether to encode option prompt and option value characters. Defaults to `true`.
      *
-     * The rest of the options will be rendered as the attributes of the resulting tag. The values will
-     * be HTML-encoded using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
+     * The rest of the options will be rendered as the attributes of the resulting tag. The values will be HTML-encoded
+     * using {@see encode()}. If a value is null, the corresponding attribute will not be rendered.
      * See {@see renderTagAttributes()} for details on how attributes are being rendered.
      *
-     * @return string the generated list box tag
+     * @return string the generated list box tag.
      */
-    public static function listBox($name, $selection = null, $items = [], $options = []): string
-    {
+    public static function listBox(
```

Comment:
It was OK before.

Reply:
PSR-2 max 120 characters for line, like the new PSR-12.